src/plugins/ssh/index.ts

```diff
@@ -292,6 +271,78 @@ const createCommand = (
   };
 };
 
+/** Add common args used by both SSH & SCP to args.sshOptions.
+ *
+ * These common args are only added if they have not been explicitly specified by the end user.
+ */
+const addCommonArgs = (args: CommandArgs, proxyCommand: string[]) => {
+  const sshOptions = args.sshOptions ? args.sshOptions : [];
```

Comment:
could we return `const sshOptions` so we can avoid having to repeat `args.sshOptions ? args.sshOptions : []`?